components/financial_assistance/app/assets/javascripts/financial_assistance/income.js

```diff
@@ -386,10 +388,20 @@ document.addEventListener("turbolinks:load", function () {
       var clonedForm = newIncomeForm.clone(true, true)
         .removeClass('hidden')
         .appendTo(incomeListEl);
+      if (incomeListEl.children().length > 1 && incomeListEl.children().first().attr('id') === 'hidden-income-form') {
+        console.log("removed by otherindex check??");
+        incomeListEl.children().first().remove();
+      }
+      if (unemploymentIndex != 0) {
+        var previousForm = clonedForm.prev('.new-unemployment-income-form');
+        console.log("removed by otherindex check??");
```

Comment:
probably should pull this little console log guy out, if you have time

Reply:
removed thanks https://github.com/ideacrew/enroll/pull/3431/commits/4430229198d3828e7d0d7ed520925f7726e64e2b